nixos/tests/qboot.nix

```diff
@@ -0,0 +1,14 @@
+import ./make-test-python.nix ({ pkgs, ...} : {
+  name = "qboot";
+
+  machine = { ... }: {
+    virtualisation.bios = pkgs.qboot;
+  };
+
+  testScript =
+    ''
+      start_all()
+      machine.wait_for_unit("multi-user.target")
+      machine.shutdown()
```

Comment:
Do we want to explicitly see that we don't hang on a shutdown, or why aren't we happy waiting for `multi-user.target` only?